src/configuration.h

```diff
@@ -397,3 +397,18 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 #define GPS_POWER_CTRL_CH 3
 #define LORA_POWER_CTRL_CH 2
+
+// -----------------------------------------------------------------------------
+// WiFi Configuration
+// -----------------------------------------------------------------------------
+//
+// Set WiFi credentials using the API (Does this work?)
+//    meshtastic --setpref WiFi_SSID_NAME yournetwork
+//    meshtastic --setpref WiFi_SSID_PASSWORD yourpassword
+//
+// WiFi_Mode
+//  0 = Disabled
+//  1 = Enabled
+#define WiFi_MODE 0
```

Comment:
looks great! but how about we make the default of WiFi_MODE be 1 and the default NAME and password be empty.  This would cause the checks to all be done at runtime.  So the user (via the python tool) could say "meshtastic --setpref wifi_ssid "mynet" etc...

This would avoid users having to edit and build their own bins.  Also, it would leave wifi off until they have set their net name, so not increasing power consumption to run wifi.

Reply:
Will do.